.github/workflows/haskell.yml

```diff
@@ -1,12 +1,6 @@
 name: Haskell CI
 
-on:
-  # Trigger the workflow on push or pull request,
-  # but only for the master branch
-  push:
-    branches:
-      - master
-  pull_request:
+on: [push]
```

Comment:
Won’t this turn CI off on PR builds?

Reply:
No, but it will run CI on every push... which might not be ideal.